manifests/qpid.pp

```diff
@@ -11,6 +11,13 @@
     Group['qpidd'] ->
     User<|title == $katello_user|>{groups +> 'qpidd'}
   }
+  exec { 'delete katello entitlements queue if bound to *.*':
+    command   => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' del queue ${candlepin_event_queue} --force",
+    onlyif    => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' list binding | grep ${candlepin_event_queue} | grep '*.*'",
+    path      => '/usr/bin',
+    require   => Service['qpidd'],
+    logoutput => true,
+  }
```

Comment:
Don't we need the notify? I thought if we deleted the queue, we should cause the other one to refresh.  Maybe if just the ordering is right it works because of the onlyif/unless on the execs.